lms/djangoapps/student_profile/views.py

```diff
@@ -120,3 +124,74 @@ def learner_profile_context(request, profile_username, user_is_staff):
         context['data']['badges_api_url'] = reverse("badges_api:user_assertions", kwargs={'username': profile_username})
 
     return context
+
+def learner_profile_statistics_context(profile_username, profile_statistics):
+    """Context for the learner student_profile/profile_statistics.html page.
+
+    Args:
+        profile_username (str): username of user whose profile is requested.
+        profile_statistics (str): html in string 
+
+    Returns:
+        dict
+    """
+    context = {
+        'data': {
+            'error_message': '',
+            'profile_statistics': profile_statistics,
+            'username': profile_username,
+            },
+    }
+
+
+    return context
+
+def learner_performance(request, course_id=None, student_id=None):
+    """
+        view for maiking request on mobytize platform
+    """
+    if student_id:
+        student = User.objects.get(id=int(student_id))
+    else:
+        student = request.user
+
+    date_from = CourseEnrollment.objects.filter(user=student.id).filter(is_active=True).order_by('created').first().created
+
+    from datetime import datetime
+    if date_from:
```

Comment:
Why is this import here? Can there be no date_from? If so, why are we using it in line 165 without any checks?